tests/dispatch_test.c

```diff
@@ -82,6 +84,35 @@ dispatch_test_check_evfilt_read_for_fd(int fd)
 #endif
 }
 
+const char *
+dispatch_test_get_test_file(void)
+{
+#if defined(_WIN32)
+	static char path[MAX_PATH];
```

Comment:
I think that you want `MAX_PATH + 1` to account for the trailing null-terminator.

Reply:
`MAX_PATH` by definition includes room for a null terminator.